src/app/services/article.service.ts

```diff
@@ -53,28 +53,22 @@ export class ArticleService {
     docRef.set({ path: fullPath, downloadUrl: url });
   }
 
-  async getLatestArticles() {
-    const articlesRef = this.fsdb.collection('articleData/articles/articles');
-    const querySnap = await articlesRef.orderBy('timestamp', 'desc').limit(12).get();
-    return this.arrayFromCollectionSnapshot(querySnap);
+  async latestArticlesRef() {
```

Comment:
Here's where we can strongly type the method return value so the component doesn't throw linter/compile errors... We'll also drop the now-incorrect async/await syntax.
```suggestion
  latestArticlesRef(): AngularFirestoreCollection<ArticleDetailFirestore> {
```